internal/clusterconfig/clusterconfig.go

```diff
@@ -40,7 +42,11 @@ const (
 
 	clusterIDFileName = "cluster-id-override.json"
 
-	idmsFileName = "image-digest-mirror-set.json"
+	idmsFileName  = "image-digest-mirror-set.json"
+	icspsFileName = "image-content-policy-list.json"
+
+	caBundleCMName   = "user-ca-bundle"
```

Comment:
This assumes the configmap name is user-ca-bundle ? Not sure we can assume that

Reply:
Actually we can. This is hardcoded name in mco too